### PR TITLE
password change command

### PR DESCRIPTION
**Pull request by boehlke**
This is required to set the admin password when creating an instance programmatically.

### CODE REVIEW

**Comment by emanuelschuetze** [comment]
Can you explain the use case for this Commander? Why you don't use the initial password?

File: openslides/core/management/commands/insecurechangepassword.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from openslides.users.models import User
+
+
+class Command(BaseCommand):
+    """
+    Command to change OpenSlides config values.
```
Comment (normanjaeckel) [inline openslides/core/management/commands/insecurechangepassword.py]: Docstring is out of date.

File: openslides/core/management/commands/insecurechangepassword.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    """
+    help = 'Changes OpenSlides config values.'
```
Comment (normanjaeckel) [inline openslides/core/management/commands/insecurechangepassword.py]: Help text is wrong.

File: openslides/core/management/commands/insecurechangepassword.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+            help='The name of the user to set the password for'
+        )
+        parser.add_argument(
+            'password',
+            help='The new password of the user'
+        )
+
+    def handle(self, *args, **options):
+        try:
+            user = User.objects.get(username=options['username'])
```
Comment (normanjaeckel) [inline openslides/core/management/commands/insecurechangepassword.py]: Please let except statement immediately follow the line which may raise the exception. The rest of the code belongs to the else block or is outside the whole try-except-else block.

File: openslides/core/management/commands/insecurechangepassword.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+            user.set_password(options['password'])
+            user.save()
+        except (Exception,) as e:
```
Comment (normanjaeckel) [inline openslides/core/management/commands/insecurechangepassword.py]: Please catch only User.DoesNotExist because only this is the expected exception.

**Comment by boehlke** [comment]
@normanjaeckel Fixed

File: openslides/core/management/commands/insecurechangepassword.py [hunk 5]
```diff
@@ -5,9 +5,9 @@
 
 class Command(BaseCommand):
     """
-    Command to change OpenSlides config values.
+    Command to change a users's password.
```
Comment (emanuelschuetze) [inline openslides/core/management/commands/insecurechangepassword.py]: typo: `..an user's ...`

**Comment by normanjaeckel** [comment]
Please rebase commits. Then ready to merge.

**Comment by emanuelschuetze** [comment]
@boehlke ping, please rebase your commits to one commit. Then I'll merge it.

**Comment by emanuelschuetze** [comment]
Rebase ok. Merge...

**Pull request merged**